deckhouse-controller/pkg/controller/module-controllers/release/controller.go

```diff
@@ -910,20 +910,20 @@ func (c *moduleReleaseReconciler) parseNotificationConfig(ctx context.Context) (
 }
 
 func validateModule(def models.DeckhouseModuleDefinition) error {
-	if def.Weight < 900 || def.Weight > 999 {
```

Comment:
Why have you removed this validation? We shouldn't pass external modules with some low weight, it can break everything

Reply:
I rechecked the option with installation without overrides. In this case, the module weight is correctly determined and validation will proceed as expected.
Returned the weight check. But we need to recheck the behavior in the case of overrides. And the case when we create the module release ourselves.